ando/engine.py

```diff
@@ -172,6 +172,36 @@ def is_AnDO(directory, verbose, webcall):
             return 0,None;
         else : return found_err
 
+def next_is_AnDO(path):
+    """
+
+    Check if file path adhere to AnDO.
+    Main method of the validator. uses other class methods for checking
+    different aspects of the directory path.
+
+
+    Args:
+        directory ([str]): [names of the directory to check]
+
+    Returns:
+        [bool]: [does the directory adhere to the ando specification]
+    """
+    paths=[]
+    conditions=[]
+    if type(path) == str:
+        for r, d, f in os.walk(path):
+            for file in f:
+                paths.append(os.path.join(r, file))
+    else:
```

Comment:
This condition here results in different results if the same path is once provided as string and once e.g. as pathlib object. Does this make sense? Maybe it would be better to either get rid of the else condition or only use this case when actually a list is provided.